src/util/depositRequest.ts

```diff
@@ -1,21 +1,16 @@
 import * as bitcoin from "bitcoinjs-lib";
-import { ECPairFactory, ECPairAPI } from "ecpair";
 
-import { scanTxOutSet } from "@/actions/bitcoinClient";
-import {
-  bytesToHex as uint8ArrayToHexString,
-  hexToBytes as hexToUint8Array,
-} from "@stacks/common";
+import { hexToBytes as hexToUint8Array } from "@stacks/common";
 
 import { Taptree } from "bitcoinjs-lib/src/types";
 
 import * as bip341 from "bitcoinjs-lib/src/payments/bip341";
 
 import ecc from "@bitcoinerlab/secp256k1";
-const ECPair: ECPairAPI = ECPairFactory(ecc);
+// const ECPair: ECPairAPI = ECPairFactory(ecc);
 bitcoin.initEccLib(ecc);
 
-const NUMS_X_COORDINATE = new Uint8Array([
+export const NUMS_X_COORDINATE = new Uint8Array([
   0x50, 0x92, 0x9b, 0x74, 0xc1, 0xa0, 0x49, 0x54, 0xb7, 0x8b, 0x4b, 0x60, 0x35,
   0xe9, 0x7a, 0x5e, 0x07, 0x8a, 0x5a, 0x0f, 0x28, 0xec, 0x96, 0xd5, 0x47, 0xbf,
   0xee, 0x9a, 0xce, 0x80, 0x3a, 0xc0,
```

Comment:
Where do these come from?

Reply:
Was supplied by @djordon

Reply:
Thanks, Daniel educated me: https://github.com/stacks-network/sbtc/blob/a3a927f759871440962d8f8066108e5b0af696a0/sbtc/src/lib.rs#L16-L33

Reply:
Yeah these should be the same ones in

https://github.com/stacks-network/sbtc/blob/a3a927f759871440962d8f8066108e5b0af696a0/sbtc/src/lib.rs#L16-L33

Those bytes are the same ones in BIP-0341, and they are computed by taking the SHA256 of the generator point of the secp256k1 elliptic curve. I think the idea is that this point is verifiably random so it is unlikely that anyone knows the private key.